### PR TITLE
TM-3732 mock pr for eas location

### PR DESCRIPTION
**Pull request by andrews-john**
😺
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2430)
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/742)

### CODE REVIEW

File: src/services/employees.js [hunk 1]
```diff
@@ -795,6 +795,11 @@ const get_v3_persons_agenda_items = async query => {
                   posnumtext: assignmentInfo.currentAssignment.currentPosition.pos_num_text,
                   posgradecode: assignmentInfo.currentAssignment.currentPosition.pos_grade_code,
                   postitledesc: assignmentInfo.currentAssignment.currentPosition.pos_title_desc,
+                  currentLocation: {
```
Comment (elizabeth-jimenez) [inline src/services/employees.js:798]: Was this confirmed to be coming through from the WS payload?

**Comment by elizabeth-jimenez** [comment]
🎊 Nice first mock ticket! 🥳

File: src/services/employees.js (hunk 1 above)
Comment (SidelineCory24) [reply src/services/employees.js:798]: will double check this

Comment (SidelineCory24) [reply src/services/employees.js:798]: Doesn't look like the data is coming through at all. This is what I'm seeing in dev1 on my end:

image

**Comment by SidelineCory24** [comment]
This PR and the [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/742) are blocked by a WS issue.

**Comment by andrews-john** [comment]
As of 2/7/23 we still dont see the currentLocation object in the payload
image

**Comment by scott062** [comment]
This is unblocked now - see updated endpoint in dev1

**Pull request closed without merging**